keyboards/ymdk/oni_hhkb/readme.md

```diff
@@ -0,0 +1,19 @@
+# YMDK ONI HHKB
+
+![ONI HHKB PCB, Hotswap Tsangan/HHKB PCB with RGB Matrix and backlight](https://imgur.com/DB87on8.jpg)
```

Comment:
This is a link to the page on Imgur - not the image.

Aside - admins usually wanted the `h`/Huge (1024×1024) variant of the file - but it seems Imgur has changed how they do that sometime this year (2025), so not sure what the crew wants to do with that.

Reply:
I don't know what the problem is here. When checking in the browser, I do see the image (I think the URL must be terminated by .jpg, which is the case). Also, the image I've linked is the only one I could find, from the YMDK website, and I'm not sure I could find a bigger one. Some directions would be appreciated here.

Reply:
`h` requirement has been removed.

Could you please update the url to `https://i.imgur.com/DB87on8.jpeg` so that is a direct link to the image.